setup.py

```diff
@@ -4,16 +4,18 @@
 setup(
     name="jgscm",
     description="Jupyter Google Cloud Storage ContentsManager",
-    version="0.1.9",
+    version="0.2.0",
```

Comment:
I think we should signify this is not the original version, but our derived one.  Also, this seems like a patch version change.  How about `0.1.10-hail`?

Reply:
The `-`syntax isn't a valid PEP 440 version number, for this kind of situation we're supposed to suffix a `+hail` which is called a ["local version identifier"](https://www.python.org/dev/peps/pep-0440/#local-version-identifiers).

Reply:
Oh, it's not semvar?  My bad?  Good to know.